lib/Notification/styles.scss

```diff
@@ -1,114 +1,13 @@
-/* ==========================================================================
-   Config
-   ========================================================================== */
-$notification-font-size: $typo-size-base !default;
-$notification-padding: $layout-spacing-base/2 !default;
-$notification-border-width: 1px !default;
-$notification-border-style: solid !default;
-
-$notification-danger-bg: #FEEEEE !default;
-$notification-danger-text: #B33D37 !default;
-$notification-danger-border: #FCCECB !default;
-
-$notification-warning-bg: #FFFCEF !default;
-$notification-warning-text: #71591C !default;
-$notification-warning-border: #FCF5D2 !default;
-
-$notification-success-bg: #EFFAF2 !default;
-$notification-success-text: #206B17 !default;
-$notification-success-border: #CFF1D8 !default;
-
-$notification-info-bg: #F7F9FA !default;
-$notification-info-text: #4B5761 !default;
-$notification-info-border: #EDEEEF !default;
-
-$notification-tease-bg: #E2D2F9 !default;
-$notification-tease-text: $primary-purple !default;
-$notification-tease-border: transparent !default;
-
-$notification-primary-text: white !default;
-
 /* ==========================================================================
    Styles
    ========================================================================== */
 .notification {
-	font-size: $notification-font-size;
-	padding: $notification-padding;
-	border: $notification-border-width $notification-border-style transparent;
-	border-radius: $border-radius-base;
-
-	&__content {
-		width: 100%;
-	}
-
+	text-shadow: none;
```

Comment:
💚 We should look to move this styling to whatever text element is rendered as the child of notification, rather than the notification element handling it as the parent.

Total future change, as I think this would cause a bucket load of work at the moment!

Reply:
🚩 Can we not add this with tailwind?

Reply:
no, its a limitation :(